pkg/cmd/root.go

```diff
@@ -0,0 +1,72 @@
+package cmd
```

Comment:
I'd maybe make this into `cmd.go`, it would be more obvious to me that that is the 'main' package file

Reply:
Ack! Will rename!